src/NewRelicServiceProvider.php

```diff
@@ -20,6 +20,15 @@ public function register()
         $this->app->singleton(Newrelic::class, function() {
             return new Newrelic();
         });
+
+        $this->mergeConfigFrom(
```

Comment:
Can you wrap this in an `if (\file_exists(__DIR__.'/../config/newRelic.php'))`? I want to make sure that if someone upgrades from 3.0.0 which doesn't have a configuration file their setups don't break all of a sudden.

After this it's good to go!

Reply:
No probs, that's been updated now :)